src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionPublishButton.tag

```diff
@@ -1,14 +1,17 @@
 <%@ tag description="instructorFeedbacks and instructorHome - Feedback Session publish/unpublish button" %>
 <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
 <%@ attribute name="publishButton" type="teammates.ui.template.FeedbackSessionPublishButton" required="true" %>
+<%@ attribute name="showTooltip" type="java.lang.Boolean" required="true" %>
 <%@ attribute name="buttonType" required="true" %>
```

Comment:
I asked you before about this [here](https://github.com/TEAMMATES/teammates/pull/7312#discussion_r117710436) and you said both the ways are fine :stuck_out_tongue_closed_eyes: